tests/inspectdb/tests.py

```diff
@@ -668,3 +668,55 @@ def test_composite_primary_key(self):
         finally:
             with connection.cursor() as cursor:
                 cursor.execute("DROP TABLE %s" % table_name)
+
+
+class TestInspecDbCrossDatabaseRelationships(TestCase):
+    databases = ["default", "other"]
```

Comment:
Move this tests to the `backends/mysql/test_introspection.py`.

Reply:
b0bd89da78f1ce7c2b68e24ccf154d0bc8a759eb